dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/configuration/JobParametersConfiguration.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.configuration;
+
+
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.boot.autoconfigure.mongo.MongoProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import uk.ac.ebi.eva.dbsnpimporter.parameters.DbsnpDatasource;
+import uk.ac.ebi.eva.dbsnpimporter.parameters.Parameters;
+
+import java.util.HashMap;
+
+@Configuration
+public class JobParametersConfiguration {
+
+    private JobParametersBuilder parametersBuilder;
+
+    @Bean
+    JobParameters jobParameters(Parameters parameters, DbsnpDatasource dbsnpDatasource,
+                                MongoProperties mongoProperties) {
+        parametersBuilder = new JobParametersBuilder();
+
+        addParameter("assembly", parameters.getAssembly());
+        addParameter("assemblyTypes", String.join(",", parameters.getAssemblyTypes()));
+        addParameter("batchId", parameters.getBatchId());
+        addParameter("chunkSize", parameters.getChunkSize());
+        addParameter("dbsnpBuild", parameters.getDbsnpBuild());
+        addParameter("pageSize", parameters.getPageSize());
+        addParameter("processor", parameters.getProcessor());
+        addParameter("variantsCollection", parameters.getVariantsCollection());
+
+        addParameter("driverClassName", dbsnpDatasource.getDriverClassName());
+        addParameter("url", dbsnpDatasource.getUrl());
+        addParameter("username", dbsnpDatasource.getUsername());
```

Comment:
Should not be this parameter a "NonIdentifying" one like _mongoUsername_? 
Also, if they are not identifying, why don't we exclude them, as we do with the passwords? (I think that excluding the passwords is a good idea). Are they useful for debugging? (if a job fails)

Reply:
agree making the dbsnp user non-identifying, for consistency's sake.

yes, mainly debugging the mongo connection. the other parameter classes are ours, so we can do a toString and show them when we are injecting the classes that use them.